client/my-sites/hosting/style.scss

```diff
@@ -12,17 +12,25 @@
 	.card {
 		display: flex;
 		flex-direction: row;
-		width: calc( 100% - 10px );
+		width: 100%;
```

Comment:
All these changes have been needed to ensure the data loss warning is aligned with the card above.